content/api/commands/mount.md

```diff
@@ -302,117 +327,17 @@ it('User profile should display user name', () => {
 })
 ```
 
+> The `getStore` method is a factory method that initializes a new Redux store.
```

Comment:
Instead of `>` use an `<Alert type="info"></Alert>`

Reply:
done in #7888195